OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	class NukePowerInfo : SupportPowerInfo, IRulesetLoaded, Requires<BodyOrientationInfo>
```

Comment:
The `IRulesetLoaded` removal isn't part of the fix, but it was redundant because we inherit from `SupportPowerInfo`.